regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -73,6 +73,10 @@ var CommentReviewView = Backbone.View.extend({
 
     this.$form.find('[name="comments"]').val(JSON.stringify(commentData));
 
+    // hide toggle elements
+    $('.toggle .collapsible').attr('aria-hidden', 'true').hide();
+    $('.toggle .toggle-button-close').attr('aria-hidden', 'true').hide();
```

Comment:
Can we avoid looking at the whole DOM? I think we can accomplish the same via a scoped query like
```javascript
this.$el.find('.toggle .collapsible').//etc.
```